x-pack/plugins/security/public/views/management/edit_role/components/privileges/kibana/space_aware_privilege_form.js

```diff
@@ -150,24 +158,40 @@ export class SpaceAwarePrivilegeForm extends Component {
               onChange={this.onExistingSpacePrivilegesChange}
             />
 
-            {(Object.keys(this.state.spacePrivileges).length > 0) && <EuiSpacer />}
+            {hasAssignedSpacePrivileges && <EuiSpacer />}
 
-            {this.props.editable && (
-              <Fragment>
-                {this.state.privilegeForms.map((form, index) => this.getSpaceForm(form, index, basePrivilege))}
-                {availableSpaces.length > 0 &&
+            {this.getSpaceForms(basePrivilege)}
+
+            <EuiFlexGroup responsive={false} alignItems={'flexEnd'}>
```

Comment:
I would leave `responsive` as `true` (or just remove this prop) and change `alignItems` to 'baseline' so the text in the button and the text link line up.

Reply:
good call, thanks!